product-service/pom.xml

```diff
@@ -12,7 +12,6 @@
 	</parent>
 	
 	<artifactId>product-service</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
```

Comment:
I know that subprojects inherit from parent pom but I left them for a reason for future changes if the services independently.

Reply:
Ah, sorry, normally I use the one repository, one version tactic and I like for my pom's to only contain the necessary information.

Reply:
We are on the same page that we like to keep things clean and just enough information, this is why I add comments to get your opinion.